src/uk/ac/open/audio/streaming/PlayerProgress.java

```diff
@@ -604,6 +625,39 @@ else if(currentState==State.PLAYING || currentState==State.RECORDING)
 		}
 	}
 
+	/**
+	 *set border to jpanel on focus
+	 **/
+	private void setFocusBorder(Graphics2D g2) {
+		g2.setColor(new Color(153,204,255));
+		g2.drawRect(2,2,getWidth()-5,getHeight()-5);
+	}
+
+	/**
+	 * Set pause time.
+	 * @param pausetime
+	 **/
+	public void setpausetime(int pausetime) {
```

Comment:
Trivia: Function names should also be in camelCase, so setPauseTime(int pauseTime)